docs/get-started/index.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Get Started
+lang: en-US
+sidebar: false
+layout: home
+hideHero: true
+---
+
+<div class="getstarted">
+  <div class="center">
+    <h1> Build on Ethereum, the Open Development Platform.
+    </h1>
+
+  <h2> Learn the basics of Ethereum with Ethereum Studio, our web based IDE where you can code and test basic applications.</h2>
+  <a href="https://studio.ethereum.org" target="blank">
+  <button class="try-button"> Try it now</button>
+  </a>
+  <div class="terminal-gif">
+  <img src="/terminal.gif" />
+  </div>
+  <h2 class="bold">
+    Start coding right away. Save many hours of development setup, try the following templates to see what you can build with Ethereum.
+  </h2>
+  <div class="features">
+    <div class="feature">
+      <div class="icon">
+        👋
+      </div>
+      <div class="box">
+        <div class="box-title">
+          Hello World
+        </div>
+        <div class="box-description">
+          Learn how to code a smart contract, how to create the simplest contract and how to interact with it by sending transactions.
+        </div>
+        <div class="box-link">
+          <a href="https://studio.ethereum.org/1">Run Hello World </a>
+        </div>
+      </div>
+    </div>
+    <div class="feature">
+      <div class="icon">
+        🗝️
+      </div>
+      <div class="box">
+        <div class="box-title">
+          ERC-20 Token
+        </div>
+        <div class="box-description">
+          Discover how you can create unique incentives by creating
```

Comment:
Being lazy here as it's line-broken…

> A template that uses the ERC-20 standard to define a reproducible token you can create and send to others.

Reply:
in this context, does `reproducible` = `fungible`? if so, I think the latter is the more common and understandable term

Reply:
I would challenge that fungible is that well known outside of our community, somewhat why I tried to use a more general word.